Upstream: under review, https://github.com/iputils/iputils/pull/177
Reason: We don't have a separate sbin anymore

From fa012d4bef0bd6ecfee5626bb015665d911bbaff Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Mon, 15 Apr 2019 20:44:28 +0200
Subject: [PATCH] Use the value of sbindir to install rinfod, rarpd & ninfod

...instead of hard-coding 'sbin'.
---
 meson.build        | 4 ++--
 ninfod/meson.build | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/meson.build b/meson.build
index efb4dc2..57b485b 100644
--- a/meson.build
+++ b/meson.build
@@ -282,7 +282,7 @@ endif
 
 if build_rinfod == true
 	executable('rdisc', ['rdisc.c', git_version_h],
-		install_dir: 'sbin',
+		install_dir: get_option('sbindir'),
 		link_with : [libcommon],
 		install: true)
 	if systemd.found()
@@ -318,7 +318,7 @@ endif
 
 if build_rarpd == true
 	executable('rarpd', ['rarpd.c', git_version_h],
-		install_dir: 'sbin',
+		install_dir: get_option('sbindir'),
 		link_with : [libcommon],
 		install: true)
 	if systemd.found()
diff --git a/ninfod/meson.build b/ninfod/meson.build
index ea7ec1b..a0ee94d 100644
--- a/ninfod/meson.build
+++ b/ninfod/meson.build
@@ -14,7 +14,7 @@ executable('ninfod', [ninfod_sources, git_version_h],
 	link_with : [libcommon],
 	include_directories : inc,
 	install: true,
-	install_dir: 'sbin')
+	install_dir: get_option('sbindir'))
 conf_data = configuration_data()
 conf_data.set('prefix', get_option('prefix'))
 
-- 
2.21.0

